boards/nxp/frdm_imx93/frdm_imx93_mimx9352_a55.dts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mimx93_a55.dtsi>
+#include "frdm_imx93-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "NXP i.MX93 A55";
+	compatible = "fsl,mimx93";
+
+	chosen {
+		zephyr,console = &lpuart2;
+		zephyr,shell-uart = &lpuart2;
+		/* sram node actually locates at DDR DRAM */
+		zephyr,sram = &dram;
+		zephyr,canbus = &flexcan2;
+	};
+
+	cpus {
+		cpu@0 {
+			status = "disabled";
+		};
+	};
+
+	dram: memory@d0000000 {
+		reg = <0xd0000000 DT_SIZE_M(1)>;
+	};
+
+	aliases {
+		led0 = &led_r;
+		led1 = &led_g;
+		sw0 = &btn_1;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_r: led_r {
+			label = "LED_R";
+			gpios = <&gpio2 13 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_g: led_g {
+			label = "LED_G";
+			gpios = <&gpio2 4 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_b: led_b {
+			label = "LED_B";
+			gpios = <&gpio2 12 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		btn_1: btn_1{
+			label = "BTN1";
+			gpios = <&gpio_exp0 5 GPIO_ACTIVE_LOW>;
+			zephyr,code = <INPUT_KEY_0>;
+		};
+
+		btn_2: btn_2{
+			label = "BTN2";
+			gpios = <&gpio_exp0 6 GPIO_ACTIVE_LOW>;
+			zephyr,code = <INPUT_KEY_1>;
+		};
+	};
+
+	can_phy0: can-phy0 {
+		compatible = "nxp,tja1051", "can-transceiver-gpio";
+		standby-gpios = <&gpio_exp0 23 GPIO_ACTIVE_HIGH>;
+		max-bitrate = <5000000>;
+		#phy-cells = <0>;
+		status = "okay";
+	};
+};
+
+&lpuart1 {
+	status = "disabled";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart1_default>;
+	pinctrl-names = "default";
+};
+
+&lpuart2 {
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart2_default>;
+	pinctrl-names = "default";
+};
+
+&lpi2c1 {
+	status = "disabled";
+	clock-frequency = <I2C_BITRATE_FAST>;
+	pinctrl-0 = <&i2c1_default>;
+	pinctrl-names = "default";
+};
+
+&lpi2c2 {
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_FAST>;
+	pinctrl-0 = <&i2c2_default>;
+	pinctrl-names = "default";
+
+	gpio_exp0: pcal6524@22 {
+		compatible = "nxp,pcal6524";
+		reg = <0x22>;
+		gpio-controller;
+		#gpio-cells = <2>;
+		ngpios = <24>;
+		int-gpios = <&gpio3 27 (GPIO_ACTIVE_LOW|GPIO_PULL_UP)>;
+		status = "okay";
+	};
+};
+
+&lpspi3 {
+	status = "disabled";
+	clock-frequency = <1000000>;
+	pinctrl-0 = <&spi3_default>;
+	pinctrl-names = "default";
+};
+
+&gpio1{
```

Comment:
space between the node label name and bracket pls. Also, isn't this already enabled in the DTSI?

Reply:
GPIOs are not enabled by default in dtsi - peripherals are all disabled. I'd like to keep this consistent.